cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -2090,6 +2092,8 @@ pub(crate) fn define(
             r#"
         Unsigned integer division by an immediate constant.
 
+        Same as `udiv`, but one operand is a sign extended 64 bit immediate constant.
```

Comment:
I don't have an opinion myself yet, but is sign-extending the best approach in this case?